src/wallet/rpc/wallet.cpp

```diff
@@ -234,8 +234,18 @@ static RPCHelpMan loadwallet()
                     }
                 },
                 RPCExamples{
-                    HelpExampleCli("loadwallet", "\"test.dat\"")
-            + HelpExampleRpc("loadwallet", "\"test.dat\"")
+                    "\nLoad regular wallet with files under wallets/foo/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo\"")
+                    + HelpExampleRpc("loadwallet", "\"foo\"")
+                    + "\nLoad regular wallet with files under wallets/foo/specialWallet/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo/specialWallet/\"")
+                    + HelpExampleRpc("loadwallet", "\"foo/specialWallet/\"")
+                    + "\nLoad wallet using absolute path:\n"
+                    + HelpExampleCli("loadwallet", "\"/Users/joe/specialWallet/\"")
+                    + HelpExampleRpc("loadwallet", "\"/Users/joe/specialWallet/\"")
```

Comment:
This should probably reflect the OS bitcoind was compiled for

Reply:
I made the change to reflect different path string depending on OS. Let me know if this is what you meant. I was also debating doing an alternative way where I have 2 example types, one MacOS and other Unix, so that web docs like https://developer.bitcoin.org/reference/rpc/loadwallet.html show both examples.

Reply:
Went ahead with 2 example types displayed.